app/assets/images/payment_icons/afterpay.svg

```diff
@@ -1 +1,13 @@
-<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-afterpay"><title id="pi-afterpay">Afterpay</title><path fill="#b2fce4" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/><path d="m27.4 6.8-2.7-1.6-2.7-1.6c-1.8-1-4.1.3-4.1 2.4v.4c0 .2.1.4.3.5l1.3.7c.4.2.8-.1.8-.5v-.8c0-.4.4-.7.8-.5l2.5 1.4 2.4 1.5c.4.2.4.7 0 .9l-2.5 1.4-2.5 1.6c-.4.2-.8-.1-.8-.5v-.4c0-2.1-2.3-3.4-4.1-2.4l-2.7 1.6-2.7 1.6c-1.8 1-1.8 3.7 0 4.7l2.7 1.6 2.7 1.6c1.8 1 4.1-.3 4.1-2.4v-.4c0-.2-.1-.4-.3-.5l-1.3-.7c-.4-.2-.8.1-.8.5v.8c0 .4-.4.7-.8.5l-2.5-1.4-2.5-1.5c-.4-.2-.4-.7 0-.9l2.5-1.4 2.5-1.4c.4-.2.8.1.8.5v.4c0 2.1 2.3 3.4 4.1 2.4l2.7-1.6 2.7-1.6c1.9-1.2 1.9-3.8.1-4.9z"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" role="img" width="38" height="24" aria-labelledby="pi-afterpay">
+<title id="pi-afterpay">Afterpay</title>
+<path opacity=".07" d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z"/>
+<path fill="#fff" d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32"/>
+<path d="M26.645,7.242l-2.47-1.463l-2.471-1.464c-1.646-0.915-3.751,0.274-3.751,2.196v0.366c0,0.183,0.092,0.366,0.274,0.458
+	l1.189,0.641c0.366,0.183,0.732-0.092,0.732-0.458V6.785c0-0.366,0.366-0.64,0.732-0.458l2.287,1.281l2.195,1.373
+	c0.365,0.183,0.365,0.64,0,0.823l-2.287,1.281l-2.287,1.464c-0.366,0.183-0.732-0.091-0.732-0.457v-0.366
+	c0-1.922-2.104-3.111-3.751-2.196l-2.47,1.464l-2.471,1.464c-1.647,0.915-1.647,3.385,0,4.3l2.471,1.464l2.47,1.464
+	c1.647,0.915,3.751-0.275,3.751-2.195v-0.365c0-0.183-0.093-0.365-0.274-0.458l-1.189-0.64c-0.366-0.183-0.732,0.092-0.732,0.458
+	v0.731c0,0.365-0.366,0.641-0.731,0.458l-2.288-1.281l-2.288-1.373c-0.366-0.183-0.366-0.641,0-0.823l2.288-1.279l2.288-1.281
+	c0.366-0.183,0.731,0.092,0.731,0.458v0.366c0,1.921,2.105,3.11,3.753,2.195l2.47-1.464l2.47-1.464
+	C28.293,10.628,28.293,8.249,26.645,7.242z"/>
+</svg>
```

Comment:
Can we please remove these changes to `afterpay.svg`?
<img width="262" alt="Screen Shot 2021-03-26 at 16 35 56" src="https://user-images.githubusercontent.com/1557529/112598051-613fd300-8e51-11eb-9801-0dac1424f067.png">
<img width="85" alt="Screen Shot 2021-03-26 at 16 36 00" src="https://user-images.githubusercontent.com/1557529/112598054-63099680-8e51-11eb-885a-405ba507d4cc.png">
Source: https://www.afterpay.com/index